options/locale/locale_de-DE.ini

```diff
@@ -326,7 +326,7 @@ hi_user_x=Hallo <b>%s</b>,
 
 activate_account=Bitte aktiviere dein Konto
 activate_account.title=%s, bitte aktiviere dein Konto
-activate_account.text_1=Hallo <b>%[1]s</b>, danke für deine Registrierung bei %[2]!
+activate_account.text_1=Hallo <b>%[1]s</b>, danke für deine Registrierung bei %[2]s!
```

Comment:
Weird.
In crowdin, both `s` are already present…
![image](https://user-images.githubusercontent.com/51889757/135355392-afe48da6-827f-4a97-95b8-3fbbeeb84044.png)

Reply:
They were probably changed after we made the RC